src/SamplesApp/UITests.Shared/Windows_UI_Xaml/UIElementTests/UIElement_InitializationSequence.xaml.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+using System;
+using System.Linq;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Uno.Disposables;
+
+namespace UITests.Windows_UI_Xaml.UIElementTests
+{
+	[Sample("UIElement", "GH Bugs", Description = "This is an illustration of GH Bug #3519")]
+	public sealed partial class UIElement_InitializationSequence : Page
+	{
+		public UIElement_InitializationSequence()
+		{
+			Instance = this;
```

Comment:
Codacy found an issue: [Remove this assignment of 'Instance' or initialize it statically.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5850270)